app.js

```diff
@@ -30,5 +30,6 @@ app.post('/zip', (req, res) => {
     }
   })
 })
-app.listen(port, () => console.log(`Service listening on port ${port}!`))
+//We don't listen anymore, just export it.
+//app.listen(port, () => console.log(`Service listening on port ${port}!`))
```

Comment:
That would break the existing functionality running in a Docker container.

Reply:
Updated the code, so it also still listens to the port

Reply:
Cool, I will test and let you know, thanks!

Reply:
I don't know how big of a task you are trying to run on the post /zip route. But keep in mind lambda functions have got a timeout (standard this is set to 3 seconds)